src/UI/CompletionsView.re

```diff
@@ -85,17 +155,60 @@ let createElement =
               |> int_of_float;
             let height =
               Types.EditorFont.getHeight(editorFont) +. 0.5 |> int_of_float;
+            let remainingWidth = 400 - width;
+
+            let detailElem =
+              switch (curr.completionDetail) {
+              | None => React.empty
+              | Some(v) when String.length(v) > 0 =>
+                let detailWidth =
+                  Types.EditorFont.measure(~text=v, editorFont)
+                  +. 0.5
+                  |> int_of_float;
+                let detailWidth = min(remainingWidth, detailWidth);
```

Comment:
Very nitty, but you could just add `|> min(remainingWidth)`  here instead of a new binding shadowing the old.

Reply:
Good catch, clearer than shadowing the name.